src/tardev-snapshotter/src/snapshotter.rs

```diff
@@ -1012,16 +1012,79 @@ impl TarDevSnapshotter {
                         return Err(anyhow!(copy_error));
                     }
                 }
+                file.flush()?;
 
-                trace!("Appending index to {:?}", &base_name);
-                file.rewind().context("failed to rewind the file handle")?;
-                tarindex::append_index(&mut file).context("failed to append tar index")?;
+                // Create an erofs metadata using mkfs.erofs
+                let layer_path = base_name.with_extension("erofs");
```

Comment:
Same concern as earlier, why are we calling this .erofs when it is erofs + merkle tree?

Reply:
Oh, I did not understand your question earlier. I get you now. I will use a better naming mechanism